src/lib/util.ts

```diff
@@ -236,11 +236,16 @@ export function schemeName(scheme: ISchemeState|IContractInfo, fallback?: string
       if (genericSchemeInfo) {
         name = genericSchemeInfo.specs.name;
       } else {
-        name = "Generic Scheme";
+        // Adding the address is a bit long for a title
+        // name = `Blockchain Interaction (${contractToCall})`;
+        name = `Blockchain Interaction`;
       }
     } else {
-      name = "Generic Scheme";
+      // this should never happen...
+      name = `Blockchain Interaction`;
     }
+  } else if (scheme.name === "ContributionReward") {
+    name ="Funding and Voting Power"
```

Comment:
sorry to say I really don't like this one either 🤷‍♂️ its pretty confusing to me. Vote on Funding would be clearer.

Reply:
What about "Asset Allocation" or "Asset Distribution"?